services/metadata/conf/main/logback.xml

```diff
@@ -11,7 +11,7 @@
   </appender>
 
 
-  <Logger name="donkey" level="debug" />
+  <Logger name="terrain" level="debug" />
```

Comment:
Copy-pasta. If you blame it, my name will show up.
I can remove those if it seems appropriate.